contracts/InterchainProposalExecutor.sol

```diff
@@ -28,7 +28,9 @@ contract InterchainProposalExecutor is IInterchainProposalExecutor, AxelarExecut
     constructor(address _gateway, address _owner) AxelarExecutable(_gateway) {
         if (_owner == address(0)) revert InvalidAddress();
 
-        transferOwnership(_owner);
+        assembly {
+            sstore(_OWNER_SLOT, _owner)
+        }
```

Comment:
Let me know if I should emit an `OwnershipTransferred` event here

Reply:
Yes, i think it's a good idea to emit event here since the ownership address is important enough to make it clear and transparent.

Reply:
i'd prefer creating an internal `_transferOwnership` method in Ownable to encapsulate this from inheriting contracts